### DISCUSSION TITLE
[Feature Request]: Add a Proxying Services

### DISCUSSION THREAD

**Discussion by maximepvrt** [beyondcode/herd-community#379] [Ideas]
### Feature Description

Same as Valet Proxying Services https://laravel.com/docs/10.x/valet#proxying-services

**Comment by mpociot** [comment]
Yeah, we'll add this as we still need to migrate from our Valet fork, to the official Laravel Valet package.

**Comment by secondmanveran** [comment]
This works out of the box with the exception of referencing a unique nginx error page for the referenced domain.

```
herd proxy mailpit http://0.0.0.0:8025 --secure
```

I just had to edit the nginx config file and change the error_log path:

```
error_log /Users/vince/Library/Application Support/Herd/Log/nginx-error.log
```
mailpit.test open like lightning.

**Comment by maximepvrt** [comment]
Hi @mpociot,

I wanted to follow up on the feature request I submitted regarding the addition of Proxying Services to Herd, which is currently available in Valet.

I saw your comment about migrating from the Valet fork to the official Laravel Valet package. Is there an estimated timeline for its implementation?

**Comment by patrickomeara** [comment]
@secondmanveran's tip helped her. The error log wasn't being created for the proxy address. 

```sh
mkdir -p "~/Library/Application Support/Herd/Log"
touch "~/Library/Application Support/Herd/Log/<proxy>.test-error.log"
```

**Comment by maximepvrt** [comment]
@mpociot The herd proxy command works great! Why is it currently undocumented and not officially supported? It would be a huge help for many of us.
